packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -778,6 +778,30 @@ let getSingleUseLegacyLogCallback = (logger: ITelemetryLoggerExt, type: string)
 	};
 };
 
+/**
+ * Load the stores from a snapshot and returns the runtime.
+ * @param params - An object housing the runtime properties:
+ * - context - Context of the container.
+ * - registryEntries - Mapping from data store types to their corresponding factories.
+ * - existing - Pass 'true' if loading from an existing snapshot.
+ * - runtimeOptions - Additional options to be passed to the runtime
+ * - containerScope - runtime services provided with context
+ * - provideEntryPoint - Promise that resolves to an object which will act as entryPoint for the Container.
+ * This object should provide all the functionality that the Container is expected to provide to the loader layer.
+ * @legacy
+ * @alpha
+ */
+export async function loadContainerRuntime(params: {
+	context: IContainerContext;
+	registryEntries: NamedFluidDataStoreRegistryEntries;
+	existing: boolean;
+	runtimeOptions?: IContainerRuntimeOptions;
+	containerScope?: FluidObject;
```

Comment:
Could we get away without passing scope in this new world?  Following the same migration strategy you mention, if folks find it to be a blocker we could work with them to figure out the alternative.

Reply:
While i think this would be a righteous thing to do, i don't have a clear vision for a replacement, and i think the class exposure causes more pain, then the scope concept, so i'd rather not bundle the two.

Reply:
For more options to consider - could also expose two free functions instead of just one, (A) with all the radical "best practices" and most-extremely-scoped return type that we recommend as the preferred option, and (B) with whatever compromises are needed to move people off the class as quickly as possible.

Reply:
i don't have a problem with something like A existing, but i don't think i'll add it here, or drive it. When i think about the future of the runtime i envision something much more composable, and very far from the world of today, and not doable with the class exposure, think a builder pattern where you pick the parts you want, datastores, gc, summarizer, blobs, id compressor etc, to get there i need to class gone, so the class itself can be decomposed, and we can explore different compositional patterns with loadContainerRuntime just continuing to be the uber composition